### PR TITLE
bump: bump up dependencies for release-1.3 branch

### PR DESCRIPTION
**Pull request by Two-Hearts**
This PR is for `release-1.3` branch.

### CODE REVIEW

File: test/e2e/suite/command/verify.go [hunk 1]
```diff
@@ -240,7 +240,7 @@ var _ = Describe("notation verify", func() {
 			notation.Exec("sign", "--timestamp-url", "http://timestamp.digicert.com", "--timestamp-root-cert", filepath.Join(NotationE2EConfigPath, "timestamp", "DigiCertTSARootSHA384.cer"), artifact.ReferenceWithDigest()).
 				MatchKeyWords(SignSuccessfully)
 
-			notation.Exec("verify", artifact.ReferenceWithDigest(), "-v").
```
Comment (shizhMSFT) [inline test/e2e/suite/command/verify.go:243]: What's this change for?

**Review by shizhMSFT — COMMENTED** [review]
Have you covered the GitHub workflow updates? They are also important for releasing the CLI binaries.

File: test/e2e/suite/command/verify.go (hunk 1 above)
Comment (Two-Hearts) [reply test/e2e/suite/command/verify.go:243]: This is due to notation-go log update. Some logs are moved from `INFO` to `DEBUG` level, hence `-v` will no longer show the expected log message.

**Comment by Two-Hearts** [comment]
> Have you covered the GitHub workflow updates? They are also important for releasing the CLI binaries.

Added.

**Review by shizhMSFT — CHANGES_REQUESTED** [review]
#1111 and #1112 seem has not been covered yet.

**Comment by Two-Hearts** [comment]
> #1111 and #1112 seem has not been covered yet.

okay. They are new PRs created yesterday and not merged into main yet. Although unlikely, if they cannot be merged into main due to some reason, we will have to create another PR to revert changes in this PR. @shizhMSFT

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by vaninrao10 — APPROVED** [review]
LGTM

**Pull request merged**